Makefile

```diff
@@ -401,3 +401,10 @@ show-img:
 
 show-container-engine:
 	@echo -n $(CONTAINER_ENGINE)
+
+.PHONY: deployment-script
+deployment-script: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
+	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
+	$(KUSTOMIZE) build config/default > rhdh-operator-${VERSION}.yaml
```

Comment:
Could you also add the `rhdh-operator-*.yaml` to `.gitignore`?
Or I'm wondering if it wouldn't make sense to also commit and push this automatically (to a file like `deploy/rhdh-operator.yaml`)? Just like we are doing with the bundle manifests..

This way, it would become even more straightforward to deploy the operator: `kubectl apply -f https://raw.githubusercontent.com/janus-idp/operator/main/deploy/rhdh-operator.yaml`
WDYT?

Reply:
I would not like to suggest deploying "random" version from git repo in common case (it is good for ones who deal with sources but not sure for everyone else). I better add it to .gitignore as you suggested.

Reply:
> I would not like to suggest deploying "random" version from git repo

Well, the point is that this would not be random, but coming from the operator repo itself, so legitimate. Rather than having to deal with cloning the sources..
But anyway, fine for now to just Git-ignore it.
I think, for the future, this file could be generated and attached as a release artifact when we publish a new GH release.